dashboard/app/models/levels/applab.rb

```diff
@@ -125,12 +129,15 @@ def update_json_fields
   end
 
   def validate_maker_if_needed
-    maker_enabled = properties['makerlab_enabled'] == 'true'
+    makerlab_property = properties['makerlab_enabled']
```

Comment:
Out of scope for this PR, but would it be difficult backwards compatibility-wise to change the name of this property to something that indicates it's not a boolean? Tricky to read with the current naming

Reply:
This is a good call. Elsewhere we use makerAPIEnabled (as in 'which' makerAPIEnabled). Does that sound like enough in the right direction?

Reply:
huh, I find that confusing also (ie, why not just `makerAPI`?). Could be a just me thing. In any case, I think fine for now, but maybe something to consider as we move to multiple APIs?